alkemio_virtual_contributor_engine/models.py

```
@@ -1,29 +1,44 @@
-    azure_deployment=env.embeddings_model_name,
-)
+logger = logging.getLogger(__name__)
+
+try:
+    openai_embeddings = AzureOpenAIEmbeddings(
+        api_key=SecretStr(env.openai_key) if env.openai_key else None,
+        api_version=env.openai_api_version,
+        azure_endpoint=env.openai_endpoint,
+        azure_deployment=env.embeddings_model_name,
+    )
+except Exception as e:
```

Ruff (0.15.4)

[warning] Do not catch blind exception: `Exception`

(BLE001)